crt_portal/static/js/complaint_view_filters.js

```diff
@@ -135,16 +135,22 @@
    * @param {Object} updates The properties of the filter state to be updated
    */
   function mutateFilterDataWithUpdates(state, updates) {
-    for (var key in updates) {
+    for (const [key, value] of Object.entries(updates)) {
```

Comment:
Breaks in IE11 with `Expected Identifier` due to the `const` (and probably the destructuring). Not sure if we want to fix this, but giving you a heads up.

Reply:
Intended, thanks for flagging 😄 https://github.com/usdoj-crt/crt-portal/pull/708